src/mjolnir/util.cc

```diff
@@ -29,5 +45,63 @@ std::string remove_double_quotes(const std::string& s) {
   return ret;
 }
 
+void build_tile_set(const boost::property_tree::ptree& config, const std::vector<std::string>& input_files, const std::string& bin_file_prefix) {
```

Comment:
i pulled this out of `valhalla_build_tiles` and put it into a function so other programs (namely tests) can build tiles for a given set of pbfs